website/docs/guides/vector-search.mdx

```diff
@@ -41,7 +39,26 @@ FerretDB supports the following vector index kinds:
 
 Vector index can be created using the usual `createIndexes` command with the following syntax:
 
-<CodeBlock language="js">{IndexSyntax}</CodeBlock>
+```js
+db.runCommand({
```

Comment:
Previously, it was in a file https://github.com/FerretDB/FerretDB/blob/459a9fd904ac97844f32a4a5b11e0397efa7b11a/website/docs/guides/requests/index-syntax.js

Why do we use inline code now?

Reply:
To not mix cts generated js files with syntax files.